ui_framework/doc_site/src/views/color_picker/color_picker_example.js

```diff
@@ -29,5 +33,19 @@ export default props => (
         <ColorPicker />
       </GuideDemo>
     </GuideSection>
+    <GuideSection
+      title="Color Picker with a label, reset link, and no color label"
```

Comment:
Can we change this to `Color Picker with label and reset link`, and c reate a separate example that shows the ColorPicker alone without the color label? This way we'll be able to see the "(transparent)" label in this example.